src/components/Settings/Notifications/NotificationsWebPush/index.tsx

```diff
@@ -0,0 +1,123 @@
+import axios from 'axios';
+import { Field, Form, Formik } from 'formik';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import { useToasts } from 'react-toast-notifications';
+import useSWR from 'swr';
+import globalMessages from '../../../../i18n/globalMessages';
+import Button from '../../../Common/Button';
+import LoadingSpinner from '../../../Common/LoadingSpinner';
+import NotificationTypeSelector from '../../../NotificationTypeSelector';
+
+const messages = defineMessages({
+  agentenabled: 'Enable Agent',
+  webhookUrl: 'Webhook URL',
+  webpushsettingssaved: 'Web Push notification settings saved successfully!',
+  webpushsettingsfailed: 'Web Push notification settings failed to save.',
+  testsent: 'Web Push test notification sent!',
```

Comment:
```suggestion
  webpushsettingssaved: 'Browser notification settings saved successfully!',
  webpushsettingsfailed: 'Browser notification settings failed to save.',
  testsent: 'Browser test notification sent!',
```
"Web Push" isn't a proper noun, so "push" shouldn't be capitalized.

That said, I don't think that "Web Push" is the best name for this agent? Tautulli just names this agent "Browser," which is better and less vague IMO.  (Also, notifications can be enabled/disabled globally by browser on Windows, and that setting would include the ones pushed by Overseerr.)

Anyway, some food for thought.  I don't think "Browser" is perfect either, but I do think it is better than "Web Push."

Reply:
Browser is not very good either I feel? It comes from the Push API (a Web API) so I felt Web Push was pretty descriptive. If there is something better then sure, but browser is not better.